src/steadystate_adjoint.jl

```diff
@@ -98,8 +100,7 @@ end
     end
 
     if !needs_jac
-        # TODO: FixedVecJacOperator should respect the `autojacvec` of the algorithm
-        operator = FixedVecJacOperator(f, y, p, Val(DiffEqBase.isinplace(sol.prob)))
+        operator = VecJac(f, y, p; autodiff = get_autodiff_from_vjp(vjp))
```

Comment:
This is not really equivalent. IIRC `VecJac` recomputes the pullback everytime a call to `mul!` is made. In this case, we have a fixed input, only the seeding changes so we compute the pullback once and just reevaluate it multiple times.

Reply:
Yes, see https://github.com/JuliaDiff/SparseDiffTools.jl/pull/245